rktlet/util/image.go

```diff
@@ -45,3 +46,21 @@ func ApplyDefaultImageTag(image string) (string, error) {
 	}
 	return image, nil
 }
+
+func GetCanonicalImageName(imageName string) (string, error) {
```

Comment:
Unit test for that function would be nice

Reply:
@nhlfr Yes exactly. I'll do that.